docker-compose.yml

```diff
@@ -6,26 +6,12 @@ services:
     image: "postgres:latest"
     environment:
       POSTGRES_USER: ${DB_USER}
-      POSTGRES_PASSWORD: ${DB_PASSWD}
+      POSTGRES_PASSWORD: ${DB_PASS}
       PGDATA: /data/postgres
     volumes:
        - ./postgres:/data/postgres
     ports:
-       - "127.0.0.1:5432:5432"
-    networks:
-      - postgres
-    restart: unless-stopped
-  
-  pgadmin:
-    container_name: pgadmin_container
-    image: dpage/pgadmin4
-    environment:
-      PGADMIN_DEFAULT_EMAIL: ${PGADMIN_EMAIL}
-      PGADMIN_DEFAULT_PASSWORD: ${PGADMIN_PASSWD}
-    volumes:
-       - ./pgadmin:/root/.pgadmin
-    ports:
-      - "5050:80"
+       - "127.0.0.1:5432:${DB_PORT}"
```

Comment:
This is wrong.
In a bind, port1:port2,
port1 in your local system will be bound to port2 in
docker's internal network stack.